src/widget/wtrackmenu.cpp

```diff
@@ -790,9 +794,11 @@ void WTrackMenu::updateMenus() {
     }
 
     if (featureIsEnabled(Feature::RemoveFromDisk)) {
-        bool locked = m_pTrackModel->hasCapabilities(TrackModel::Capability::Locked);
-        if (m_pTrackModel->hasCapabilities(TrackModel::Capability::RemoveFromDisk)) {
-            m_pRemoveFromDiskAct->setEnabled(!locked);
+        if (m_pTrackModel) {
```

Comment:
Why is this check needed here but not for any other option?

Reply:
Based on the assumption outlined above:
either track loaded or track model (and track refs loaded), so if the menu is initialized from a track widget then there's no track model. > segfault

Reply:
Please add a source code comment about it.